python/ray/actor.py

```diff
@@ -594,31 +594,47 @@ def method(self):
             # runtime_env specified in the @ray.remote decorator.
             new_runtime_env = None
 
+        cls_options = dict(
+            num_cpus=num_cpus,
+            num_gpus=num_gpus,
+            memory=memory,
+            object_store_memory=object_store_memory,
+            resources=resources,
+            accelerator_type=accelerator_type,
+            max_concurrency=max_concurrency,
+            max_restarts=max_restarts,
+            max_task_retries=max_task_retries,
+            name=name,
+            namespace=namespace,
+            lifetime=lifetime,
+            placement_group=placement_group,
+            placement_group_bundle_index=placement_group_bundle_index,
+            placement_group_capture_child_tasks=(placement_group_capture_child_tasks),
+            runtime_env=new_runtime_env,
+            max_pending_calls=max_pending_calls,
+            scheduling_strategy=scheduling_strategy,
+        )
+
         class ActorOptionWrapper:
             def remote(self, *args, **kwargs):
                 return actor_cls._remote(
                     args=args,
                     kwargs=kwargs,
-                    num_cpus=num_cpus,
-                    num_gpus=num_gpus,
-                    memory=memory,
-                    object_store_memory=object_store_memory,
-                    resources=resources,
-                    accelerator_type=accelerator_type,
-                    max_concurrency=max_concurrency,
-                    max_restarts=max_restarts,
-                    max_task_retries=max_task_retries,
-                    name=name,
-                    namespace=namespace,
-                    lifetime=lifetime,
-                    placement_group=placement_group,
-                    placement_group_bundle_index=placement_group_bundle_index,
-                    placement_group_capture_child_tasks=(
-                        placement_group_capture_child_tasks
-                    ),
-                    runtime_env=new_runtime_env,
-                    max_pending_calls=max_pending_calls,
-                    scheduling_strategy=scheduling_strategy,
+                    **cls_options,
+                )
+
+            @DeveloperAPI
```

Comment:
Could we keep this as experimental for now?

Reply:
sure, i didn't find "experimental" in `annotations.py` tho, what's the best practice in ray to mark it as experimental ?

Reply:
How about not annotating it and marking it `**Experimental**` in docstring?

Reply:
sure :)